src/vivarium_nih_us_cvd/components/observers.py

```diff
@@ -402,38 +403,60 @@ class PAFObserver(Component):
     def configuration_defaults(self) -> Dict[str, Dict]:
         return {
             "stratification": {
-                f"{self.risk.name}_paf_on_{self.target.name}": self.CONFIGURATION_DEFAULTS[
+                f"joint_paf_on_{self.target.name}": self.CONFIGURATION_DEFAULTS[
                     "stratification"
                 ]["paf"]
             }
         }
 
-    def __init__(self, risk: str, target: str):
+    def __init__(self, target: str):
         super().__init__()
-        self.risk = EntityString(risk)
         self.target = TargetString(target)
-
-    # noinspection PyAttributeOutsideInit
-    def setup(self, builder: Builder) -> None:
-        self.risk_effect = builder.components.get_component(
-            f"paf_calculation_risk_effect.{self.risk}.{self.target}"
+        self.risks_and_mediators = self._get_risks_and_mediators()
+
+    def _get_risks_and_mediators(self) -> Set[str]:
+        """ MEDIATOR_NAMES is a nested dict like {risk: {target: [mediators]}}.
+        For each target in MEDIATOR_NAMES (the second-level keys), extract the
```

Comment:
Nit: this comment isn't quite right, as it's not for each target - it's for this specific target.